frontend/workflows/experimentation/src/start-experiment.jsx

```diff
@@ -0,0 +1,209 @@
+import React from "react";
+import { Button, client, Confirmation, MetadataTable, useWizardContext } from "@clutch-sh/core";
+import { useDataLayout } from "@clutch-sh/data-layout";
+import { Wizard, WizardStep } from "@clutch-sh/wizard";
+import { Grid } from "@material-ui/core";
+import * as yup from "yup";
+
+const ClusterPairTargetDetails = () => {
+  const { onSubmit } = useWizardContext();
+  const clusterPairData = useDataLayout("clusterPairTargetData");
+  const clusterPair = clusterPairData.displayValue();
+  const update = (key, value) => {
+    clusterPairData.updateData(key, value);
+  };
+
+  return (
+    <WizardStep error={clusterPairData.error}>
+      <MetadataTable
+        onUpdate={update}
+        data={[
+          {
+            name: "Downstream Cluster",
+            value: clusterPair.downstreamCluster,
+            input: {
+              type: "string",
+              key: "downstreamCluster",
+            },
+          },
+          {
+            name: "Upstream Cluster",
+            value: clusterPair.upstreamCluster,
+            input: {
+              type: "string",
+              key: "upstreamCluster",
+            },
+          },
+        ]}
+      />
+      <Grid container justify="center">
+        <Button text="Next" destructive onClick={onSubmit} />
+      </Grid>
+    </WizardStep>
+  );
+};
+
+const AbortExperimentDetails = () => {
+  const { onSubmit, onBack } = useWizardContext();
+  const abortExperimentData = useDataLayout("abortExperimentData");
+  const abortExperiment = abortExperimentData.displayValue();
```

Comment:
Digging into a potential issue with using `displayValue`. Let's use value for now.

```suggestion
  const abortExperiment = abortExperimentData.value;
```